frigate/config.py

```diff
@@ -46,6 +46,9 @@ class DetectorConfig(FrigateBaseModel):
 
 class UIConfig(FrigateBaseModel):
     use_experimental: bool = Field(default=False, title="Experimental UI")
+    show_confirmation_prompts: bool = Field(
```

Comment:
With this change, you'll want to add this field to the docs as well https://github.com/blakeblackshear/frigate/blob/release-0.11.0/docs/docs/configuration/user_interface.md

Reply:
Thanks. Updated.